src/Framework/Numerical/MathUtils.h

```diff
@@ -128,6 +129,9 @@ namespace math
   double NonNegative    (double x);
   double NonNegative    (float  x);
 
+  TLorentzVector GetParallel(const TLorentzVector lv, const TVector3 unitVec);
+  TLorentzVector GetOrthogonal(const TLorentzVector lv, const TVector3 unitVec);
```

Comment:
If the inputs are `const`, make them references or don't make them `const` at all if they are copies. 
I prefer the const reference version